modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/tests

```diff
@@ -29,4 +29,24 @@
     design = 'INSFVScalarFieldAdvection.md'
     issues = '#16732'
   []
+  [momentum-outflow-bcs]
+    type = 'Exodiff'
+    input = 2d-rc-no-slip-outflow-bcs.i
+    exodiff = 2d-rc-no-slip-outflow-bcs_out.e
```

Comment:
can you use the exodus created by the other one (the one with flux kernels)?

Reply:
Yes, and I do. `gold/2d-rc-no-slip-outflow-bcs_out.e` is just a symlink to the flux kernel result. Doing it this way prevents any chance for race conditions in the input directory